en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -186,29 +186,28 @@ all_videos <- read_csv(video_files, col_names = FALSE, id = "videoId", show_col_
   select(videoId, videoChannelTitle = channelTitle, videoTitle = title, commentCount)
 ```
 
-To confirm channel titles, and the number of comments per channel, simply input `all_videos` to print the results. Finally, run the following code to join the video and comment data:
+To confirm channel titles and the number of comments per channel, simply input `all_videos` to print the results. Finally, run the following code to join the video and comment data:
 
 ```
 all_data <- inner_join(all_comments, all_videos)
 count(all_data, sort(videoChannelTitle))
 ```
 
-Alternatively, if you would like to utilize our sample data, you can download it from the Github repository. Use this code to read in the sample data: 
+Alternatively, if you are using our sample data, you can read it in with the following code: 
 
 ```
 all_data <- read.csv("ytdt_data/all_data.csv")
 ```
 
-You may also choose to use a YouTube comment dataset downloaded with a tool other than YouTube Data Tools. However, if you do, you’ll first need to ensure it is formatted the same way that data downloaded with YouTube Data Tools would be by reordering and re-naming columns.
+You may also choose to use a YouTube comment dataset downloaded with a tool other than YouTube Data Tools. However, if you do, you’ll first need to ensure it is formatted the same way by reordering and re-naming its columns.
 
-## Data Labeling
-If you are using the data we’ve provided, you will also need to add a [partisan indicator](LINK to definition) to the dataset. 
+### Data Labeling
 
-Our case study project investigates  comment discourse across left- and right-leaning video channels, so this partisan indicator allows us to later visualize and compare how video comments are [scaled](LINK to definition) based on the ideological leanings and political affiliation of each video’s creator.
+If you are using the data we’ve provided (either through Youtube Data Tools or our sample dataset), you will also need to add a 'partisan indicator' to the dataset: our case study project investigates comment discourse across left- and right-leaning video channels, so this partisan indicator allows us to later visualize and compare where video comments are positioned on the ideological scale, based on the political affiliation of each video’s creator.

```

Comment:
Is this correct? I was not sure what the 'data we've provided' referred to. If this is only the 'sample data', I can make that change.

Reply:
yeah it's the sample data, we definitely need to clarify this element throughout the lesson

Reply:
OK, I've suggested this instead:

"If you are using the data taken from the Youtube video IDs we’ve chosen for this lesson (whether you've downloaded it using Youtube Data Tools or our sample dataset), you will also need to add a 'partisan indicator':"